cmd/policy/describe.go

```diff
@@ -23,8 +23,8 @@ cloudquery policy describe github.com/COMMUNITY_GITHUB_ORG/aws
 `
 )
 
-func newCmdPolicyDescribe() *cobra.Command {
-	describePolicyCmd := &cobra.Command{
+func describe() *cobra.Command {
```

Comment:
why the change to the naming convention?

Reply:
reverted